db/migrate/20170726230852_fill_reactions_counter_cache.rb

```diff
@@ -0,0 +1,7 @@
+class FillReactionsCounterCache < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def change
+    User.all.each { |user| User.reset_counters(user.id, :media_reactions) }
```

Comment:
You probably wanna use `find_each` in this scenario because `each` will load all Users into RAM at once, while `find_each` loads them in batches of 1000.

Alternatively, the `CounterCacheResets` module in `lib/counter_cache_resets.rb` can actually do this a gajillion times faster with a zany sequence of generated SQL statements